tests/clients/init/errors.robot

```diff
@@ -204,3 +204,11 @@ See if voms-proxy-init -pwstdin fails correctly when no password is provided
     Should contain  ${output}  No credentials found!
   END
   [Teardown]  Stop using certificate
+
+See if suspended users can get a proxy
+  [Tags]  legacy
+  [Setup]  Use certificate  test2
+  ${output}  Create proxy failure  -voms ${vo1}
+  ${expected}  Set Variable If  ${vo1_legacy_voms}  User is currently suspended!  User 'CN=test2,O=IGI,C=IT, CN=Test CA,O=IGI,C=IT' is not active.
```

Comment:
```suggestion
  ${expected}  Set Variable If  ${vo1_legacy_voms}  User is currently suspended!   is not active.
```
I wouldn't put the DN, so that we don't need to change the test if the subject changes in the future.